src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -83,6 +84,20 @@ public void updatePerson(Person target, Person editedPerson) {
         indicateAddressBookChanged();
     }
 
+
+    //=========== AddGroup / RemoveGroup =====================================================================
+
+    @Override
+    public void addGroup(Person target, Tag newGroup) {
```

Comment:
A bit confusing here, I see there is a `Group` class, should the newGroup be `Group` data type instead? Same for removeGroup